### DISCUSSION TITLE
How to get feedback on a PR

### DISCUSSION THREAD

**Discussion by paulvanharen** [mu-editor/mu#2512] [Q&A]
Last year, I've tried to make a contribution to Mu, and issued a pull request (#2433). I haven't had any feedback on this PR ever since: no rejects, no requests for improvements, no explanations. 

What is the process to get this PR vetted?

**Comment by satmandu** [comment]
@carlosperate Can one request a review for PRs in this PR? What's culturally appropriate to not step on any toes?
